sql/api/query.go

```diff
@@ -12,6 +12,7 @@ type Query struct {
 	Name           string            `json:"name"`
 	Description    string            `json:"description"`
 	Query          string            `json:"query"`
+	// Deprecated: Use databricks_job resource to schedule a Query
```

Comment:
this won't have any effect, please use `SchemaPath` to set `Deprecated` flag to true in the relevant `StructToSchema` blocks

https://github.com/databricks/terraform-provider-databricks/blob/bd05be25f14fb66bfadbd30c020ade23f891baf8/pools/resource_instance_pool.go#L176-L182

Reply:
Thanks, I had https://zchee.github.io/golang-wiki/Deprecated/ in mind when adding `// Deprecated:`. Will take a look at `SchemaPath`